From 01dd1e29c607f37b7d67d362cf14b2264cf25e0e Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:39 -0400
Subject: [redhat] kabi: Adding symbol try_module_get (kernel/module.c)

Message-id: <40886cbea11cf4c2b977a5fdafe1f0a020705bb1.1410510773.git.poros@redhat.com>
Patchwork-id: 96003
O-Subject: [PATCH RHEL7.1 219/233] kabi: Adding symbol try_module_get (kernel/module.c)
Bugzilla: 1062361
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to check/protect against its own module removal.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/try_module_get
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/try_module_get
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/try_module_get

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/try_module_get b/redhat/kabi/kabi-module/kabi_ppc64/try_module_get
new file mode 100644
index 0000000..b2863a2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/try_module_get
@@ -0,0 +1,2 @@
+#1-
+0x125d0917 try_module_get vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/try_module_get b/redhat/kabi/kabi-module/kabi_s390x/try_module_get
new file mode 100644
index 0000000..22db432
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/try_module_get
@@ -0,0 +1,2 @@
+#1-
+0x3926a455 try_module_get vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/try_module_get b/redhat/kabi/kabi-module/kabi_x86_64/try_module_get
new file mode 100644
index 0000000..f5079b0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/try_module_get
@@ -0,0 +1,2 @@
+#1-
+0x84c274f9 try_module_get vmlinux EXPORT_SYMBOL
-- 
1.7.1